.bashrc

```diff
@@ -0,0 +1,30 @@
+
+if [[ $OSTYPE == "linux-gnu" && $CLOUD_SHELL == true ]]; then 
+	if [ -f "/google/devshell/bashrc.google" ]; then
+		source "/google/devshell/bashrc.google"
+	fi
+
+	cd $HOME/anthos-workshop
+
+	export BASE_DIR=$PWD
+	export WORK_DIR=$BASE_DIR/workdir
+	export PATH=$PATH:$WORK_DIR/bin:
+	export PROJECT=$(gcloud config get-value project)
+	export ISTIO_VERSION=1.1.15
+	export KUBECTX_VERSION=v0.7.0
+	export HELM_VERSION=v2.14.3
+	export GKE_CLUSTER_VERSION=1.13.7
+	export KOPS_VERSION=1.12.3
+
+	## Install tree
+	## Note: This is here and not in install-tools.sh to ensure it is available across sessions since this is installed using apt-get
+	if command -v tree 2>/dev/null; then
+		echo "tree already installed."
+	else
+		echo "Installing tree..."
+		sudo apt-get install tree
+	fi
+
+else
+    echo "This has only been tested in GCP Cloud Shell.  Only Linux (debian) is supported".
+fi
```

Comment:
Amina, quick question about the .bashrc, what was the rationale of creating a new bashrc file instead of keeping the env file doing something like the export below? Also, how would are you planing in adding this to the bashrc from the cloud shell instance? 
echo 'export ISTIO_VERSION=1.1.15' >> ~/.bashrc

Reply:
We discussed this on the post-mortem call. We want something to persist across Cloud Shell restarts. The current instructions that include sourcing env will not achieve that. 
I will update the doc to include copying ".bashrc" to home dir and sourcing it. That will guarantee consecutive restarts of the shell will populate env variables as needed.

Reply:
I didn't read your comment very well before responding. We can do what you are suggesting as well. I checked the docs on .bashrc for cloud shell and there is nothing in there that needs to stay especially in the context of the lab and that is why I decided replacing the existing one was the direction I will go with.

Reply:
Understood, but I think keeping the env file might still be simpler. We cannot forget that this source code can also be used by anyone that wants to test the lab and replacing the bashrc might overwrite things that was not intended by the user (any other variables that they added for personal use). What do you think?

Reply:
Yup you are right. I will update the env file instead then.